src/emojis.ts

```diff
@@ -1,7 +1,8 @@
 import { emojis as compressed } from './emojis.generated';
 import { decompressFromBase64LZString } from './system/string';
 
-const emojiRegex = /:([-+_a-z0-9]+):/g;
+// use lookbehind & lookahead to only match standalone emojis
+const emojiRegex = /(?<=^|\s):([-+_a-z0-9]+):(?=$|\s)/g;
```

Comment:
Unfortunately we can't use [lookbehind assertions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Lookbehind_assertion) yet as Safari only supports them in >= 16.4 (which is only a year old)

Reply:
i'll fix it tomorrow, but can i ask why a vscode extension needs support for safari?

Reply:
fixed - not the cleanest code but i think it's fine